.github/workflows/build-irods-centos.yml

```diff
@@ -1,5 +1,7 @@
 name: build-irods-centos
 on: [push, pull_request]
+env:
+   ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: true
```

Comment:
Let's add a comment which points to the documentation explaining what this is.

Reply:
Done